### PR TITLE
Nosql Status update 

### PR DESCRIPTION
**Pull request by otaviojava**
## Specification PR template
When creating a specification project release review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of wombat-spec-x.y.pdf
- [x] Specification HTML in the form of wombat-spec-x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md

### CODE REVIEW

File: nosql/1.0/_index.md [hunk 1]
```diff
@@ -16,45 +16,28 @@ https://github.com/eclipse-ee4j/nosql/pull/55
...
+  * [jakarta.nosql.mapping:mapping-core:jar:1.0.0-b3](https://repo1.maven.org/maven2/jakarta/nosql/mapping/mapping-core/1.0.0-b3/)
+  * [jakarta.nosql.mapping:mapping-key-value:jar:1.0.0-b3](https://repo1.maven.org/maven2/jakarta/nosql/mapping/mapping-key-value/1.0.0-b3/)
+  * [jakarta.nosql.mapping:mapping-column:jar:1.0.0-b3](https://repo1.maven.org/maven2/jakarta/nosql/mapping/mapping-column/1.0.0-b3/)
+  * [jakarta.nosql.mapping:mapping-document:jar:1.0.0-b3](https://repo1.maven.org/maven2/jakarta/nosql/mapping/mapping-document/1.0.0-b3/)
+  * [jakarta.nosql.mapping:mapping-api:jar:1.0.0-b3](https://repo1.maven.org/maven2/jakarta/nosql/mapping/mapping-api/1.0.0-b3/)
 
 # Compatible Implementations
 
 * [Eclipse JNoSQL](http://www.jnosql.org/)
-
```
Comment (ivargrimstad) [inline nosql/1.0/_index.md:39]: The review records should not be removed. Please add this back in.

**Review by ivargrimstad — CHANGES_REQUESTED** [review]

File: nosql/1.0/_index.md (hunk 1 above)
Comment (otaviojava) [reply nosql/1.0/_index.md]: @ivargrimstad done, thank you

**Comment by otaviojava** [comment]
@ivargrimstad ?

**Comment by ivargrimstad** [comment]
@otaviojava Please clarify if you are requesting a progress review. If that is the case, check the [Guide to the Jakarta EE Specification Process](https://jakarta.ee/committees/specification/guide/) and update the PR accordingly.

**Comment by otaviojava** [comment]
@ivargrimstad it is a milestone

**Review by ivargrimstad — APPROVED** [review]

**Comment by ivargrimstad** [comment]
As discussed on the spec committee call today, this is good to go.

**Pull request merged**